index.html

```diff
@@ -1741,7 +1975,6 @@ <h4 class="modal-title">Help</h4>
                         return;
                     }
                     var url = window.location.host;
-                    if (url.indexOf("playtak") > -1)
```

Comment:
That is required if you want to run the server locally. If you want offline client to connect to server, you could do port-forwarding. So don't remove that line in the final pull request.